contrib/extractor/CMakeLists.txt

```diff
@@ -23,24 +23,27 @@ include_directories (
      ../../src/game/Maps
 )
 
-add_executable (mapextractor
+add_executable (MapExtractor
     dbcfile.cpp
     System.cpp
     loadlib/loadlib.cpp 
     loadlib/adt.cpp
     loadlib/wdt.cpp
     )
 
-SET_TARGET_PROPERTIES (mapextractor PROPERTIES FOLDER Extractors)
+SET_TARGET_PROPERTIES (MapExtractor PROPERTIES FOLDER Extractors)
 
 if(UNIX)
   find_package(ZLIB REQUIRED)
   if(ZLIB_FOUND)
     include_directories(${ZLIB_INCLUDE_DIRS})
-    target_link_libraries(mapextractor ${ZLIB_LIBRARIES} libmpq bz2)
+    target_link_libraries(MapExtractor ${ZLIB_LIBRARIES} libmpq bz2)
   endif(ZLIB_FOUND)
 else()
-  target_link_libraries (mapextractor zlib libmpq bz2)
+  target_link_libraries (MapExtractor zlib libmpq bz2)
 endif(UNIX)
 
-install(TARGETS mapextractor DESTINATION ${BIN_DIR})
+install(TARGETS MapExtractor DESTINATION ${BIN_DIR}/Extractors)
+if(WIN32 AND MSVC)
+  install(FILES ${CMAKE_SOURCE_DIR}/bin/Debug/MapExtractor.pdb DESTINATION ${BIN_DIR}/Extractors CONFIGURATIONS Debug)
```

Comment:
`BIN_DIR` is correct (which is the install path) but you have to adjust the source, use `${CMAKE_BINARY_DIR}` for that.